go-controller/pkg/types/const.go

```diff
@@ -213,4 +213,8 @@ const (
 	// entry for the gateway routers. After this time, the entry is removed and
 	// may be refreshed with a new ARP request.
 	GRMACBindingAgeThreshold = "300"
+
+	// UserDefinedNetworkMasqueradeIPBase define the base to calculate udn
+	// masquerade IPs
+	UserDefinedNetworkMasqueradeIPBase = 10
```

Comment:
AFAICT this constant is not going to be used everywhere in the code, its special to the allocator udn alone I'd move it to being a const in that same file with ips.go